packages/package_proteowizard_3_0_9016/tool_dependencies.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="proteowizard" version="3.0.9016">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">https://teamcity.labkey.org/guestAuth/repository/download/bt17/279368:id/pwiz-bin-linux-x86_64-gcc48-release-3_0_9016.tar.bz2</action>
```

Comment:
It's not stable. Needs a mirror. I can't think of a better approach than your github one, although I don't like having to clone the huge repo to add files. :/